src/components/Tree/TreeItem.tsx

```diff
@@ -0,0 +1,92 @@
+import classnames from 'classnames';
+import React, { useEffect, useRef, useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
Instead of uuid, can you follow the getId pattern found here https://github.com/appfolio/react-gears/pull/1027/files